storage/src/remote/sock_ctrl_msg.rs

```diff
@@ -0,0 +1,753 @@
+use vmm_sys_util::errno::{Error as SysError, Result as SysResult};
```

Comment:
Copy from https://github.com/xujihui1985/db-micro-http/blob/feat/cross-platform-support/src/common/sock_ctrl_msg.rs. It should pub new crate.